D1/deliverable1.md

```diff
@@ -195,19 +262,23 @@ This plan does not address several important but separate issues raised by this
 
 These and other secondary issues will be addressed at a later stage.
 
+#### Use BedMachine as baseline
+
+Same as for Antarctica: updated BedMachine incorporating all connected ice and peripheral glaciers.
```

Comment:
Add sentence explicitly pointing out only changing BedMachine metadata? And that the goal is not to reflect the "real" ice sheet domain (which is the goal of the GEUS product)?